README.md

```diff
@@ -59,7 +59,7 @@ Please follow the steps below if you want to submit a support ticket from the Az
 8. Select the appropriate topic from the list
 9. Follow the remaining steps in the form to create the support ticket
 
-**Note:** We will continue to support issues submitted on GitHub
+**Note:** As of July 24, 2023, we no longer support issues submitted on GitHub.
```

Comment:
I suggest to remove this line altogether. The decision to stall support issues was taken long time back in the past.